packages/vetra/index.html

```diff
@@ -19,6 +19,10 @@
   </head>
   <body>
     <div id="root"></div>
-    <script type="module" src="./connect.tsx"></script>
+    <script type="module">
+      import { AppLoader } from "@powerhousedao/connect";
+      import { createRoot } from "react-dom/client";
+      createRoot(document.getElementById("root")).render(AppLoader());
```

Comment:
I'm not 100% sure about this, I think this can break some hooks according to the docs. I think we should use `React.createElement(AppLoader)` instead of `AppLoader()`. this is the recommended way to render a react component without JSX